docs/guides/HelloWorld.md

```diff
@@ -0,0 +1,66 @@
+# Hello ggez! 👋
+
+We're going to create a very simple program using `ggez`.
+It will say "Hello ggez!" and the current delta time each frame.
+
+## Prerequisites ⚠
+
+### Rust
+
+[Rust](https://www.rust-lang.org) will need to be installed.
+Follow Rust's guides for installing the latest stable version.
+
+### SDL
+
+`ggez` requires the [`SDL2` crate](https://github.com/Rust-SDL2/rust-sdl2#user-content-requirements) to be setup on your computer.
+Follow the `SDL2` crate's guide for installing the crate.
+
+## Project Setup 🛠
+
+### Cargo Crate
+
+Open up your terminal and make sure it is in the location where you want to create your project folder.
+
+```bash
+cargo new --bin hello_ggez
+cd hello_ggez
+```
+
+Open up `Cargo.toml` with a text editor and add `ggez` as a dependency:
+
+```toml
+[package]
+name = "hello_ggez"
+version = "0.1.0"
+# Replace with your name and email
+authors = ["Awesome Person awesome@person.com"]
+
+[dependencies]
+ggez = "0.4"
+```
+
+### Check Project Setup ✔
+
+Test to make sure everything is correct by running `cargo run`.
+
+Be patient. This can take a while on the first run. Subsequent builds will be much faster.
+Be sure it says `Hello, world!` at the bottom. before continuing.
```

Comment:
```diff
- at the bottom. before continuing.
+ at the bottom, before continuing.
```
I'm not much into English grammar, but it feels like there's no need in comma at all.

Reply:
Probably doesn't need any punctuation at all there. 😄